website/docs/tutorials/09-interacting-with-starknet/03-declare.md

```diff
@@ -0,0 +1,26 @@
+# Declaring new contracts
```

Comment:
1. Where is `05-deploy-account`?
2. There are two `09-*` pages: interacting with starknet & migrations

Reply:
1. i moved it back to 05
2. done